src/renderer/wg_engine/tvgWgGeometry.cpp

```diff
@@ -65,7 +65,7 @@ void WgPolyline::appendPoint(WgPoint pt)
 {
     if (pts.count > 0) {
         float distance = pts.last().dist(pt);
-        if (distance > 0) {
+        if (!WgMath::zero(distance)) {
```

Comment:
@SergeyLebedkin Please keep your original proposal:
` if (!tvg::zero(distance * 1e-1)) {`

If possible, you can add a comment with this special handling 'reason' there.